modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

```diff
@@ -60,6 +60,11 @@ Be aware that the `_<userid>_` placeholder is evaluated into a 36 character long
 ====
 
 WARNING: For strategies where creating new {orch-namespace} is needed, make sure that `che` ServiceAccount has enough permissions to do so. With OpenShift OAuth, the authenticated User must have privileges to create new {orch-namespace}.
+`self-provisioner` cluster role should be linked to  `system:authenticated:oauth` group.
+
+WARNING: In situations when  `che` ServiceAccount has not enough permissions creating new {orch-namespace} or with OpenShift OAuth
```

Comment:
I suggest restructuring this sentence slightly, smth like:
``
WARNING: Yout must use <<pre-creating-namespace>> if `che` ServiceAccount does not have enought permissions when creating new {orch-namespace}, with OpenShift OAuth when a `self-provisioner` cluster role is not linked to the`system:authenticated:oauth` group, or in other situations when {prod-short} cannot create namespaces.``

Reply:
Thx Yana, I will find a way how to put the info in the original context.

```
WARNING: Use <<pre-creating-namespace>> when:

* `che` ServiceAccount does not have enought permissions when creating new {orch-namespace}

 * OpenShift OAuth with a `self-provisioner` cluster role is not linked to the`system:authenticated:oauth` group
 
* {prod-short} cannot create namespaces
```

Reply:
done